src/stores/FreshNewStore.test.ts

```diff
@@ -0,0 +1,179 @@
+
+    const { result } = renderHook(() => ({
+      actions: useFreshDataActions(),
+      configOptions: useConfigOptions(),
+    }));
+
+    await act(async () => {
+      await result.current.actions.appFetchesConfigOptions();
+    });
+
+    expect(mockedFetchAvailableConfigs).toHaveBeenCalledOnce();
+    expect(result.current.configOptions).toEqual(mockAvailableConfigOptions);
+  });
+
+  it('should set configOptions to empty array when fetch fails', async () => {
+    mockedFetchAvailableConfigs.mockRejectedValueOnce(new Error('Network error'));
+
+    const { result } = renderHook(() => ({
+      actions: useFreshDataActions(),
+      configOptions: useConfigOptions(),
+    }));
+
+    await act(async () => {
+      await result.current.actions.appFetchesConfigOptions();
+    });
+
+    expect(mockedFetchAvailableConfigs).toHaveBeenCalledOnce();
+    expect(result.current.configOptions).toEqual([]);
+  });
+});
+
+describe('userSelectsConfig', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+  it('should update all store states on success', async () => {
+    mockedFetchConfig.mockResolvedValueOnce({
+      config: mockFreshConfigFile,
+      termsData: mockTermsData,
+    });
+
+    const { result } = renderHook(() => ({
+      actions: useFreshDataActions(),
+      config: useConfig(),
+      standardizedVariables: useStandardizedVariables(),
+      standardizedTerms: useStandardizedTerms(),
+      standardizedFormats: useStandardizedFormats(),
+    }));
+
+    await act(async () => {
+      await result.current.actions.userSelectsConfig('Neurobagel');
+    });
+
+    expect(mockedFetchConfig).toHaveBeenCalledWith('Neurobagel');
+    expect(result.current.config).toBe('Neurobagel');
+    expect(result.current.standardizedVariables).toEqual(mockFreshStandardizedVariables);
+    expect(result.current.standardizedTerms).toEqual(mockFreshStandardizedTerms);
+    expect(result.current.standardizedFormats).toEqual(mockFreshStandardizedFormats);
+  });
+
+  it('should correctly map standardized variables with all properties', async () => {
+    mockedFetchConfig.mockResolvedValueOnce({
+      config: mockFreshConfigFile,
+      termsData: mockTermsData,
+    });
+
+    const { result } = renderHook(() => ({
+      actions: useFreshDataActions(),
+      standardizedVariables: useStandardizedVariables(),
+    }));
+
+    await act(async () => {
+      await result.current.actions.userSelectsConfig('Neurobagel');
+    });
+
+    const ageVariable = result.current.standardizedVariables['nb:Age'];
+    expect(ageVariable).toBeDefined();
+    expect(ageVariable.id).toBe('nb:Age');
+    expect(ageVariable.name).toBe('Age');
+    expect(ageVariable.variable_type).toBe('Continuous');
+    expect(ageVariable.description).toBe('The age of the participant.');
+    expect(ageVariable.required).toBe(false);
+  });
+
+  it('should correctly map terms from multiple vocabularies', async () => {
```

Comment:
```suggestion
  it('should correctly load standardized terms into state', async () => {
```
not sure I understand the multiple variables part